draft-song-dns-wireformat-http-01.txt

```diff
@@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
 
    Proxy-DNS-Transport: xyz
```

Comment:
With the text it is mandatory, but I went ahead and made it extra clear. I went ahead and defined the server-side behavior if it is missing, too (HTTP response code 400).